gdpr/README.md

```diff
@@ -0,0 +1,111 @@
+# GDPR and Data Privacy Guide
+
+> Parts of this section are based on an article, 
+[A Developer’s Guide to GDPR that won’t make you sweat](https://medium.com/ft-product-technology/a-developers-guide-to-gdpr-that-won-t-make-you-sweat-4f1f7f1d9c8b) by [Mags Allen
+](https://medium.com/@mags_40745).
+
+> More reading on how we handle GDPR as Django 
+Girls Foundation can be found on our website's
+ [Terms and Conditions](https://djangogirls.org/terms-conditions/)
+ and [Cookie and Privacy Policy](https://djangogirls.org/privacy-cookies/) 
+ sections.
+
+**The General Data Protection Regulation (GDPR) is a
+ set of rules that came into effect on May 25th, 2018.** 
+ 
+It regulates the use of data by organisations.
+It applies to any organisation that stores or processes
+ any data relating to individuals inside the European 
+ Union (EU). 
+ 
+ Since we have events in Europe, it is important 
+ for Django Girls Foundation and organizers across 
+ the globe to comply with GDPR as it has legal 
+ implications. We therefore recommend all organizers 
```

Comment:
I wonder if "recommend" is not too "soft" word. Should we write instead: All organizers should comply with GDPR

Reply:
Noted.